tests/pyccel/test_pyccel.py

```diff
@@ -280,6 +244,7 @@ def test_relative_imports_in_project(language):
     compare_pyth_fort_output(pyth_out, fort_out)
 
 #------------------------------------------------------------------------------
+@pytest.mark.xdist_incompatible
 def test_absolute_imports_in_project(language):
```

Comment:
I have the same question about this test

Reply:
See above

Reply:
Fixed in 74065a32